Makefile

```diff
@@ -36,13 +36,7 @@ root-dir := $(dir $(mkfile_path))
 
 support_verilator_4 := $(shell ($(verilator) --version | grep '4\.') > /dev/null 2>&1 ; echo $$?)
 ifeq ($(support_verilator_4), 0)
-	ifndef verilator_threads
```

Comment:
Why is this removed, isn't it working correctly?

Reply:
In 32-bit configuration, tests in riscv-tests and riscv-compliance test suites are randomly stucked. Several executions of the same test are stopping at different places. Stopping means no more instruction logged in the trace but `cycles` still incremented.